ParisBNF/abb/BNFabb107.xml

```diff
@@ -29,10 +29,214 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+                        </msItem>
+                        <msItem xml:id="ms_i3">
+                            <locus from="2va" to="5vb"/>
+                            <title ref="LIT5926PsalmBE"/>
+                            <textLang mainLang="gez"/>
+                            <incipit xml:lang="gez"><hi rend="rubric">ይትባረክ፡ እግዚአብሔር፡ አምላከ፡ እስራኤ</hi>ል፡ አምላከ፡ ለኵሉ፡ ዘሥጋ፡ ወለኵሉ፡ መንፈስ፡  ሊቃነ፡ መላእክት፡ ሚካኤል፡ ወገብርኤል፡ ሱራፌል፡ ወኪሩቤል፡ </incipit>
+                            <explicit xml:lang="gez">አልቦ፡ ዘይትማሰሎ፡ እምቅድመ፡ ህሊናሁ፡ የአምር፡ ኵሉ፡ ዘይቄድስዎ፡ ሥልጣኑ፡ ዘኢየኃልቅ፡ እግዚአ፡ ኵሉ፡ ፍጥረት፡ አሜን።</explicit>
+                            <listBibl type="text">
+                                <bibl>
+                                    <ptr target="bm:Aescoly1951Recueil"/>
+                                    <citedRange unit="page">158-169</citedRange>
+                                </bibl>
+                            </listBibl>
+                        </msItem>
+                        <msItem xml:id="ms_i4">
+                            <locus from="6ra" to="9rb"/>
+                            <title ref="LIT5927PrayerBE"/>
+                            <textLang mainLang="gez"/>
+                            <incipit xml:lang="gez"><hi rend="rubric">ይትባረክ፡ እግዚአብሔ</hi>ር፡ አምላከ፡ እስራኤል፡ አ<hi rend="rubric">ምላከ፡ ለኵሉ፡ ዘነፍስ፡</hi> ወለኵሉ፡ ዘሥጋ፡ ወዝንቱ፡ ውእቱ፡ ስምየ፡ ዘነገርክዎ፡ ለርኂብ፡ ዘይነብብ፡ አፉየ፡
+                        </incipit>
+                            <explicit xml:lang="gez">ብፁዕ፡ ውእቱ፡ ዘተናዘዘ፡ ለዛቲ፡ ጸሎት፡ ዘይቤ፡ መኑ፡ ዝንቱ፡ ዘኢይሄሊ፡ በልቡ፡ ወኢይክሉ፡ ገቢረ፡ ኑፋቄ፡ የዓሥዮ፡ እግዚአብሔር።</explicit>
+                            <listBibl type="text">
+                                <bibl>
+                                    <ptr target="bm:Aescoly1951Recueil"/>
+                                    <citedRange unit="page">176-185</citedRange>
+                                </bibl>
+                            </listBibl>
+                        </msItem>
+                        <msItem xml:id="ms_i5">
+                            <locus from="9rb" to="10ra"/>
+                            <title ref="LIT5928IncantationBE"/>
+                            <textLang mainLang="gez"/>
+                            <incipit xml:lang="gez"><hi rend="rubric">ይትባረክ፡ እግዚአብሔ</hi>ር፡ አምላከ፡ እስራኤል፡ ኤ<hi rend="rubric">ል፡ ኤል፡ ኤል፡ ኤል፡ ኤል፡</hi> ኤል፡ ኤል፡ ኪናኪር፡  
+                            </incipit>
+                            <explicit xml:lang="gez">ወለኵሎሙ፡ መናፍስት፡ ርኩሳን፡ ከመ፡ ኢይቅረቡ፡ ኀበ፡ ነፍሱ፡ ወሥጋሁ፡ ለገብርከ። <hi rend="rubric">ባይገባ፡</hi> </explicit>
+                            <listBibl type="text">
+                                <bibl>
+                                    <ptr target="bm:Aescoly1951Recueil"/>
+                                    <citedRange unit="page">192-193</citedRange>
+                                </bibl>
+                            </listBibl>
+                        </msItem>
+                        <msItem xml:id="ms_i6">
+                            <locus target="#10rava"/>
```

Comment:
```
    <locus target="#10ra"/>
    <locus target="#10va"/>
```
?
I think this folio notation is not clear to me. Where exactly is the text located?

Reply:
f 10